src/components/value.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+import * as AppPropTypes from "../app-prop-types"
+
+
+const Value = React.createClass({
+  propTypes: {
+    children: AppPropTypes.value.isRequired,
+  },
+  render() {
+    const {children} = this.props
+    return <samp>{String(children)}</samp>
+  }
+})
+
+
+export default Value
```

Comment:
In this case, the `<Value>` component abstracts the transformation to string of a value (see [this line](https://github.com/openfisca/legislation-explorer/pull/75/files/fc53ba691d28d409e1d2b2288501baa46626383d#diff-8454535cc87db40d599490dc56ac44b4R12)). I would not like to repeat the `<samp>{String(value)}</samp>` code.

I think here we can use the [stateless functional components syntax](https://facebook.github.io/react/blog/2015/10/07/react-v0.14.html#stateless-functional-components) (or even a normal factory function but that's not classic).

From a conceptual point of view a component is just a function and "propTypes" are type validation about function parameters.

By the way [propTypes are deprecated](https://facebook.github.io/react/blog/#new-deprecation-warnings) since React 15.5 but I don't want to upgrade now.